.template/addons/bootstrap/stylesheets/bootstrap/bootstrap.scss.tt

```diff
@@ -0,0 +1,42 @@
+// By default every component is imported
+// But DO NOT import the whole framework but instead
+// pick what the project requires
+// and comment out the rest.
+
+@import 'bootstrap/scss/functions';
```

Comment:
Does it work @rafayet-monon, as I remember we have to mention the `node_modules` folder here as well, correct me if I'm wrong

Reply:
@andyduong1920 I created a new app from this branch It does work. In the image below you can see I used `row` and `col` from bootstrap. And the styles from bootstrap as well -

![image](https://user-images.githubusercontent.com/14927672/95569999-246d5480-0a48-11eb-9288-b57aace9459c.png)

Reply:
I think we don't need to mention the `node_modules` anymore is because we add it to the assets path in `initializer/assets.rb`

Reply:
Nice @rafayet-monon Thanks for the double-checking 🙌